cmd/helpers/config/config.go

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+package config
+
+import (
+	. "github.com/microsoft/go-sqlcmd/cmd/sqlconfig"
+)
+
+var config Sqlconfig
+
+func Update(
```

Comment:
>unc Update( [](http://example.com/codeflow?start=1&length=11)

why is this adding 3 different kinds of objects at the same time? Shouldn't they be managed independently?

Reply:
I've renamed the method to AddContextWithContainer.  These are all the input parameters needed to add a context which has an endpoint which has a docker container which is running SQL with basic auth on an endpoint.  This is a lot of params for a function, Scott Meyers would probably suggest adding an "Info" class here to carry the parameters.  Not sure if that's overkill for this single call.